slides/08_Data_Fetching.html

```diff
@@ -281,6 +281,45 @@ <h2>Exercise</h2>
               https://pokeapi.co/api/v2/pokemon/bulbasaur/
             </a>
           </section>
+          <section>
+            <p>Poke-API DTOs</p>
+            <small>pokeApi.ts</small>
+            <pre
+              style="font-size: small"
+            ><code class="hljs tsx" data-trim><script type="text/template">
```

Comment:
**Question:** Does this fit the page? Maybe as an alternative, just add the link to the github repo with the right branch. Also we could simplify / unify the types a bit, the code below is maybe optimal due to no redundancies, abut hard to guess, which one is to use ;)

Reply:
Doublechecked, it fits the page. So all good :)